### DISCUSSION TITLE
Comskip with Comcut/Comchap

### DISCUSSION THREAD

**Discussion by obert0** [dfigus/addon-tvheadend#115] [Ideas]
Hello @dfigus, I see a project on a comskip dockerfile that use comchap/comskip executable file with comskip edl file to modify the video (add chapters or remove commercials). Can you add those files? I think it is a nice addition to this addon. I personally not try it, but I will soon when I will get my tuner to work constantly. https://github.com/BrettSheleski/comchap/tree/master.

**Comment by dfigus** [comment]
Hi @obert0,
thanks for the feature request 😄 . See #133 . Once the checks are completed I will merge the PR and it will be available in an Edge build. On Sunday I would probably create a new release anyway to release the latest TVH stuff.
